zoo-mysql-dal/src/dal/projectDAL.ts

```
@@ -0,0 +1,128 @@
+            name: string
+        }
+    ): Promise<Project> {
+        try {
+            const [result] = await pool.query<ResultSetHeader>(
+                `INSERT INTO projects
+                 SET name       = ?,
+                     created_at = NOW(),
+                     updated_at = NOW()`,
+                [name]
+            );
+            return this.getProjectById({id: result.insertId});
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)